### ISSUE TITLE
[Bug]: `package.json` reqiures the wrong node version

### ISSUE THREAD

**Issue by fourlastor** [homebridge-eufy-security/plugin#44]
### What happened?

The [required Node version](https://github.com/homebridge-eufy-security/plugin/blob/master/package.json#L15) in the package.json is `>=14.17.0` right now.

However, [this line](https://github.com/homebridge-eufy-security/plugin/blob/master/homebridge-ui/server.js#L107) is using [local nullish assignment](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Logical_nullish_assignment#browser_compatibility) which is supported from node v15+ (`??=` operator).

Another possible solution (I'd go for this one) could be not to use that particular operator and keep the supported Node version to 14

See #41 for more details (I copied the log output here)

### Device Type

Indoor Camera (Supported)

### Plugin Version

v2.0.x (Supported)

### HomeBridge Version

v1.4.x (Supported)

### NodeJS Version

v14 (Supported)

### Operating System

Raspbian 10+ (Supported)

### Relevant log output

```shell
[5/27/2022, 9:52:57 AM] [Homebridge UI] Running Command: npm install -g homebridge-eufy-security@latest
[5/27/2022, 9:53:16 AM] [Homebridge UI] [homebridge-eufy-security] /usr/local/lib/node_modules/homebridge-eufy-security/homebridge-ui/server.js:107
      this.config['country'] = body.country ??= 'US';
                                            ^^^

SyntaxError: Unexpected token '??='
    at wrapSafe (internal/modules/cjs/loader.js:988:16)
    at Module._compile (internal/modules/cjs/loader.js:1036:27)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1101:10)
    at Module.load (internal/modules/cjs/loader.js:937:32)
    at Function.Module._load (internal/modules/cjs/loader.js:778:12)
    at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:76:12)
    at internal/main/run_main_module.js:17:47
[5/27/2022, 9:53:16 AM] [Homebridge UI] [homebridge-eufy-security] Child process ended
[5/27/2022, 9:53:22 AM] [Homebridge UI] Changes to config.json saved.
[5/27/2022, 9:53:22 AM] [Homebridge UI] [homebridge-eufy-security] Terminating child process...
```

**Comment by thieren** [comment]
Should be fixed in #45

**Comment by thieren** [comment]
Fixed in 2.0.1-beta.1
